src/com/pentalog/pentastagiu/homework2/problem2/Customer.java

```diff
@@ -0,0 +1,27 @@
+package com.pentalog.pentastagiu.homework2.problem2;
+
+enum membershipStatus{
```

Comment:
si la enumuri, conventia e tot UpperCase, deci ar trebui sa redenumesti in MembershipStatus